### PR TITLE
LKE-5661: Update license

### PR DESCRIPTION
**Pull request by gmasclet**
https://linkurious.atlassian.net/browse/LKE-5661

_As an admin, I expect to update my expired license_

### CODE REVIEW

File: src/api/License/index.ts [hunk 1]
```diff
@@ -6,25 +6,45 @@
...
       url: '/license',
       method: 'POST',
       params: params
     });
   }
+
+  /**
+   * Replace the saved license with the submitted one. Fails if submitted license is invalid.
+   */
+  public updateLicense(this: Request<SaveLicenseResponse>, params: ISaveLicenseParams) {
```
Comment (aminerhanemi) [inline src/api/License/index.ts:42]: I think it is better to return LicenseInfo here

**Review by aminerhanemi — CHANGES_REQUESTED** [review]

File: src/api/License/index.ts (hunk 1 above)
Comment (aminerhanemi) [reply src/api/License/index.ts:42]: not blocking (not sure if it is possible)

Comment (gmasclet) [reply src/api/License/index.ts:42]: Discussed all together: OK to implement Amine's proposal, this avoids the frontend sending an additional request to retrieve the updated `endDate` value.

Comment (gmasclet) [reply src/api/License/index.ts]: Done :ok_hand:

**Review by aminerhanemi — APPROVED** [review]

**Pull request merged**